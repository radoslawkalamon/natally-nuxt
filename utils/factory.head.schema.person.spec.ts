import { factoryHeadSchemaPerson } from '@/utils/factory.head.schema.person'

describe('Utils / Factory / Head / Schema / Person', () => {
  test('shall match snapshot', () => {
    expect(factoryHeadSchemaPerson()).toMatchSnapshot()
  })
})
