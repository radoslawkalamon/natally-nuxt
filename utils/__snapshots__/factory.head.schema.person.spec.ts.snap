// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utils / Factory / Head / Schema / Person shall match snapshot 1`] = `
{
  "@id": "https://169cm.pl/#person/radoslaw-kalamon",
  "@type": "Person",
  "name": "Rados≈Çaw Kalamon",
  "sameAs": [
    "https://radoslawkalamon.pl",
    "https://instagram.com/radoslawkalamon/",
    "https://twitter.com/radoslawkalamon",
  ],
}
`;
